modules/dateTime.js

```diff
@@ -0,0 +1,12 @@
+import { DateTime } from './luxon.js';
+
+const today = document.getElementById('date');
+const setTime = () => {
+  const now = DateTime.now();
+  const date = now.toLocaleString(DateTime.DATETIME_MED_WITH_SECONDS);
+  today.textContent = date;
+};
+
+setInterval(setTime, 1000);
+// eslint-disable-next-line import/prefer-default-export
+export { setTime };
```

Comment:
- [ ] Kindly remove the `eslint-disable` and fix the linters. The linters are there for a purpose of preventing bugs and they should not be avoided.